packages/itwinui-variables/src/themes/light.scss

```diff
@@ -4,38 +4,73 @@
 @use '../utils/hex' as *;
 @use '../utils/get-hsl' as *;
 
-$base-colors: (
-  // Background colors - default
-  background-1: hsl(0 0% 100%),
-  background-2: hsl(220 27% 98%),
-  background-3: hsl(216 17% 94%),
-  background-4: hsl(206 11% 88%),
-  background-5: hsl(210 10% 80%),
-  background-1-hover: hsl(0 0% 95%),
-  //
-  // Background colors - statuses subtle
-  background-subtle-primary: hsl(202 73% 94%),
-  background-subtle-positive: hsl(94 41% 93%),
-  background-subtle-warning: hsl(34 85% 95%),
-  background-subtle-negative: hsl(0 67% 94%),
-  //
-  // Foreground colors - default
-  foreground-1: hsl(0 0% 0%),
-  foreground-2: hsl(0 0% 20%),
-  foreground-3: hsl(0 0% 40%),
-  foreground-4: hsl(0 0% 60%),
-  foreground-5: hsl(0 0% 80%),
-  //
-  // Foreground colors - statuses
-  foreground-primary: hsl(203 100% 44%),
-  foreground-positive: hsl(95 72% 37%),
-  foreground-warning: hsl(33 89% 51%),
-  foreground-negative: hsl(0 91% 43%),
-  foreground-primary-hover: hsl(203 100% 34%),
-  foreground-positive-hover: hsl(95 72% 27%),
-  foreground-warning-hover: hsl(33 89% 41%),
-  foreground-negative-hover: hsl(0 91% 33%)
-);
+// #region SASS variables
+$bg-hue: 203;
+$bg-saturation: 10%; // More saturated than high contrast theme.
+$bg-lightness-gap: 3%;
+$fg-lightness-gap: 7%;
+// #endregion
```

Comment:
Same as other comment - maybe add a warning about how changing the above values could negatively impact contrast scores?